amundsen_application/static/js/components/TableDetail/FrequentUsers/index.tsx

```diff
@@ -35,7 +35,6 @@ export function renderReader(
     >
       <Link
         className="avatar-overlap"
-        id="frequent-users"
```

Comment:
We need this id for logging purposes. https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/ducks/utilMethods.ts#L43

Reply:
Thanks for the heads up!!

I will use a data attribute instead, as these are repeating ids on several elements.